src/main/java/org/broadinstitute/consent/http/service/TDRService.java

```diff
@@ -33,4 +38,18 @@ public ApprovedUsers getApprovedUsersForDataset(Dataset dataset) {
 
         return new ApprovedUsers(approvedUsers);
     }
+
+    public List<Integer> getDatasetIdsByIdentifier(List<String> identifiers) {
+        List<Integer> datasetIds = identifiers
```

Comment:
You can inline this too:
```suggestion
        return identifiers
```